### ISSUE TITLE
'Reaptcha' cannot be used as a JSX component.

### ISSUE THREAD

**Issue by jcsunran** [sarneeh/reaptcha#236]
'Reaptcha' cannot be used as a JSX component.
Its instance type 'Reaptcha' is not a valid JSX element.
Type 'Reaptcha' is missing the following properties from type 'ElementClass': context, setState, forceUpdate, props, refs

It works before. but today it throws this error.

**Comment by suvankarsatpati1** [comment]
Also getting same error
 TS2786:
 'Reaptcha' cannot be used as a JSX component.
  Its instance type 'Reaptcha' is not a valid JSX element.
  Type 'Reaptcha' is missing the following properties from type 'ElementClass': context, setState, forceUpdate, props, refs

   "version": "1.8.0"

**Comment by sarneeh** [comment]
I must have break something in the latest bundle improvements. Please rollback to `1.7.3` for now, I'll update here when I fix this.

**Comment by sarneeh** [comment]
@jcsunran @suvankarsatpati1 Could you create a reproduction repository? I cannot reproduce this issue.

**Comment by sarneeh** [comment]
Oh, my bad. I just found out it's a typings issue. I'll try to solve it.

**Comment by suvankarsatpati1** [comment]
```
import Reaptcha from 'reaptcha'
<form
 action={someAction}
 ...
>
  <Reaptcha             
    sitekey={superKey}
    className={styles.myclass}
   />
</form>
```

**Comment by jonasmarco** [comment]
the same here, waiting for a solution...

**Comment by sarneeh** [comment]
Fixed in `1.9.0`
